src/error.ts

```diff
@@ -9,20 +9,23 @@ declare global {
     bundle?: Bundle;
     suberrors?: Error[];
     external?: any;
-
+    imtExceptionHash?: string; // moved from imt-blueprint - it is the same thing as hash, it is just a historical inconsistency
+    imtInternalError?: Error; // moved from imt-blueprint
```

Comment:
Basically this is used just in blueprint I guess to distinguish distinguish: "file not loaded" from node and "Failed to load package" that is the actual intent. Imho it is a bad idea and if I was designing it, I would just translate the error to the "Failed to load package" and add the "file not loaded" to the "cause", but decided to keep it here for backwards compatibilty